tracee-rules/signature.go

```diff
@@ -90,6 +89,10 @@ func findRegoSigs(target string, partialEval bool, dir string) ([]types.Signatur
 			return nil
 		}
 
+		if !aioEnabled && isAIO(d.Name()) {
```

Comment:
What is this condition for? Don't we load aio.rego as embedded resource?

Reply:
Good catch. This is cruft from an older implementation I had. I've removed this and updated tests accordingly.

Reply:
https://github.com/simar7/tracee/commit/f7bedd3265e437459106b059a65572002914032c